src/app/gui.html

```diff
@@ -719,6 +719,9 @@ <h2 class="subtitle">These functions may be experimental, or have only a rudimen
 					<ul>
 						<li><a id="aValidate">Validate definition</a> using <span v-if="!window.intelligentBackend"><a href="http://openapi-converter.herokuapp.com/">OpenAPI-converter</a> (online version of <a href="https://github.com/mermade/swagger2openapi">swagger2openapi</a>)</span><span v-if="window.intelligentBackend">built-in validator</span></li>
 						<li><a id="aShinola">Generate documentation</a> using <a href="https://github.com/mermade/shinola">shinola</a> (online <a href="https://github.com/mermade/widdershins">widdershins</a> and <a href="https://github.com/mermade/shins">shins</a> pipeline)</li>
+						<li><a id="aShins" href="/shins">Preview documentation</a> using <a href="https://github.com/mermade/widdershins">widdershins</a> and <a href="https://github.com/mermade/shins">shins</a> </li>
```

Comment:
These endpoints won't be available on github pages or when running the `index.html` file locally. You need to only include them if `window.intelligentBackend` is truthy. The `<li>` tag should have a property `v-if="window.intelligentBackend"`.

Reply:
Fixed.